index.md

```diff
@@ -3,4 +3,16 @@ layout: default
 title: Home
 ---
 
-Content coming soon.
+Tackling engineering design problems using tools from convex optimization and computer science.
+
+### Recent posts
+<div>
```

Comment:
maybe with `class="posts"` just for consistency, since I just realized that's done elsewhere.